packages/pharos/src/components/image-card/pharos-image-card.scss

```diff
@@ -24,6 +25,52 @@
   min-width: 0;
 }
 
+.card__selected {
+  background-color: var(--pharos-color-white);
+  outline: solid 2px var(--pharos-color-interactive-secondary);
+  box-sizing: border-box;
+  transition: background-color var(--pharos-transition-duration-short) ease-in-out;
+
+  slot[name='image']::slotted(*) {
+    height: calc(100% - 2 * var(--pharos-spacing-three-quarters-x));
+    width: calc(100% - 2 * var(--pharos-spacing-three-quarters-x));
+    padding: var(--pharos-spacing-three-quarters-x);
+  }
+}
```

Comment:
This is currently being used as a bit of a utility class, which doesn't follow BEM very well; consider creating e.g. a `.card__link--collection.card__link--selected` and `.card__link--image.card__link--selected` split of this.

Reply:
The BEM philosophy takes a different approach: classes should be used to describe component structure and variations. Some of these variations may have the same _outcome_, but are achieved through applying styles in the context of the specific component. One way to achieve both what I'm saying and what you're saying is to use a [mixin](https://sass-lang.com/documentation/at-rules/mixin) or a [placeholder/extension](https://thesassway.com/understanding-placeholder-selectors/) to reduce the repeated CSS.

Reply:
Thanks for the explanation. The mixin is working nice.